usecases/nodes/handler.go

```diff
@@ -40,11 +41,16 @@ func NewManager(logger logrus.FieldLogger, authorizer authorizer,
 	return &Manager{logger, authorizer, db, schemaManager}
 }
 
+// GetNodeStatus aggregates the status across all nodes. It will try for a
+// maximum of 60 seconds, then timeout
 func (m *Manager) GetNodeStatus(ctx context.Context,
 	principal *models.Principal, className string, verbosity string,
 ) ([]*models.NodeStatus, error) {
+	ctxWithTimeout, cancel := context.WithTimeout(ctx, 10*time.Second)
```

Comment:
should we make this timeout value configurable? or is it easy to add it to our config the same way as we added `MODULES_CLIENT_TIMEOUT`? if not then maybe we should make a const out of it, just to make it more visible in the code

Reply:
Good point. This also highlights that I left 10s in which was not intended. I wanted to use 60s, but reduced it to 10 for testing. 

I think an env variable just for the nodes API is a bit too specific, but I think we also can't use one variable for nodes API requests, because for a batch it might be feasible to take longer than 10 or 60s. 

Maybe a constant in the code would be the best middle ground?

Reply:
Yes, I'm OK with constant, so that it will be discoverable in the code quicker

Reply:
I'll add it and also set a higher value, I'm torn what would be a good one. 60s might be too high, 10s too low. Maybe 30s?